geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/sortedset/AbstractZAddIntegrationTest.java

```diff
@@ -160,6 +160,40 @@ public void zaddCountsOnlyNewMembers_givenMultipleCopiesOfTheSameMember_toAnExis
     assertThat(jedis.zscore(SORTED_SET_KEY, "member")).isEqualTo(3.0);
   }
 
+  @Test
+  public void zaddCountsNewMemberAndExistingMemberChanges_whenCHSpecified() {
```

Comment:
Would it be possible to break this test up into two separate cases, one which tests that the CH option works for updated members, and one which tests that it works with new members? At unit and integration test level, it's generally better to try and test one thing and one thing only per test. I suppose there could also be a third case testing that specifying CH does not return anything when no members are updated or added, but maybe that behaviour could be included in an existing test case for zadd with no new members specified.